### PR TITLE
zh: update task config for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
- remove unnecessary information in task-configuration-file.md for 2.0
- update task-configuration-file-full.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: 
https://github.com/GMHDBJD/docs-dm/blob/task-config-file-2.0/zh/task-configuration-file.md
https://github.com/GMHDBJD/docs-dm/blob/task-config-file-2.0/zh/task-configuration-file-full.md

### CODE REVIEW

File: zh/task-configuration-file-full.md [hunk 1]
```diff
@@ -143,6 +164,7 @@ mysql-instances:
 | :------------ | :--------------------------------------- |
 | `routes` | 上游和下游表之间的路由 table routing 规则集。如果上游与下游的库名、表名一致，则不需要配置该项。使用场景及示例配置参见 [Table Routing](feature-overview.md#table-routing) |
 | `filters` | 上游数据库实例匹配的表的 binlog event filter 规则集。如果不需要对 binlog 进行过滤，则不需要配置该项。使用场景及示例配置参见 [Binlog Event Filter](feature-overview.md#binlog-event-filter) |
+| `column-mappings` | 对表的列值进行修改的规则集。由于 `column-mapping` 的使用限制较多，我们不推荐使用 `column-mapping` 功能作为首选方案。使用场景及示例配置参见 [Column mapping](feature-overview.md#column-mapping) |
```
Comment (toutdesuite) [inline zh/task-configuration-file-full.md:167]:
```suggestion
| `column-mappings` | 对表的列值进行修改的规则集。由于 Column mapping 的使用限制较多，不推荐使用 Column mapping 功能作为首选方案。使用场景及示例配置参见 [Column mapping](feature-overview.md#column-mapping) |
```

**Comment by toutdesuite** [comment]
@yikeke PTAL. Thanks~

File: zh/task-configuration-file-full.md [hunk 2]
```diff
@@ -25,9 +25,15 @@ category: reference
 name: test                      # 任务名称，需要全局唯一
 task-mode: all                  # 任务模式，可设为 "full"、"incremental"、"all"
 is-sharding: true               # 是否为分库分表合并任务
+shard-mode: "optimistic"        # shard DDL 同步模式，可设为 ""、"pessimistic"、"optimistic"
+ignore-checking-items: []       # 忽略的检测项，可包含 "all"、"dump_privilege"、"replication_privilege"、"version"、"binlog_enable"、"binlog_format"、"binlog_row_image"、"table_schema"、"schema_of_shard_tables"、"auto_increment_ID"
 meta-schema: "dm_meta"          # 下游储存 `meta` 信息的数据库
 remove-meta: false              # 是否在任务同步开始前移除该任务名对应的 `meta`（`checkpoint` 和 `onlineddl` 等）。
 enable-heartbeat: false         # 是否开启 `heartbeat` 功能
+heartbeat-update-interval: 1    # `heartbeat` 更新间隔
+heartbeat-report-interval: 10   # `heartbead` 反应间隔
```
Comment (yikeke) [inline zh/task-configuration-file-full.md:34]:
```suggestion
heartbeat-report-interval: 10   # `heartbeat` 反应间隔
```

File: zh/task-configuration-file-full.md [hunk 3]
```diff
@@ -60,6 +65,22 @@ filters:                                        # 上游数据库实例匹配的
...
+​    target-column: "id"                   # 将 `source-column` 修改后的值赋值给 target-column
+​    arguments: ["1", "test", "t", "_"]    # 参数列表
+  rule-2:
+​    schema-pattern: "test_*"
+​    table-pattern: "t_*"
+​    expression: "partition id"
+​    source-column: "id"
+​    target-column: "id"
+​    arguments: ["2", "test", "t", "_"]
+
```
Comment (WangXiangUSTC) [inline zh/task-configuration-file-full.md:83]: column-mappings 功能我们已经不维护了，建议删掉这部分配置

File: zh/task-configuration-file-full.md [hunk 4]
```diff
@@ -79,7 +100,7 @@ mydumpers:                           # mydumper 处理单元运行配置参数
     threads: 4                       # mydumper 从上游数据库实例导出数据的线程数量，默认值为 4
     chunk-filesize: 64               # mydumper 生成的数据文件大小，默认值为 64，单位为 MB
     skip-tz-utc: true                # 忽略对时间类型数据进行时区转化，默认值为 true
-    extra-args: "--no-locks"         # mydumper 的其他参数，在 v1.0.2 版本中 DM 会自动生成 table-list 配置，在其之前的版本仍然需要人工配置
+    extra-args: "--no-locks"         # mydumper 的其他参数
```
Comment (WangXiangUSTC) [inline zh/task-configuration-file-full.md:103]: 这里可以说明下 `不需要在 extra-args 中配置  table-list，DM 会自动生成 `

File: zh/task-configuration-file-full.md [hunk 5]
```diff
@@ -143,6 +164,7 @@ mysql-instances:
 | :------------ | :--------------------------------------- |
 | `routes` | 上游和下游表之间的路由 table routing 规则集。如果上游与下游的库名、表名一致，则不需要配置该项。使用场景及示例配置参见 [Table Routing](feature-overview.md#table-routing) |
 | `filters` | 上游数据库实例匹配的表的 binlog event filter 规则集。如果不需要对 binlog 进行过滤，则不需要配置该项。使用场景及示例配置参见 [Binlog Event Filter](feature-overview.md#binlog-event-filter) |
+| `column-mappings` | 对表的列值进行修改的规则集。由于 Column mapping 的使用限制较多，不推荐使用 Column mapping 功能作为首选方案。使用场景及示例配置参见 [Column mapping](feature-overview.md#column-mapping) |
```
Comment (WangXiangUSTC) [inline zh/task-configuration-file-full.md:167]: 可以删掉

**Pull request merged**